edx_rbac/utils.py

```diff
@@ -131,17 +133,18 @@ def create_role_auth_claim_for_user(user):
     """
     role_auth_claim = []
     for system_role_class in settings.SYSTEM_WIDE_ROLE_CLASSES:
-        app_name, model_name = system_role_class.split('.')
-        model_class = apps.get_model(app_name, model_name)
-
-        role_assignments = model_class.objects.filter(
-            user=user
-        ).select_related('role')
-
-        for role_assignment in role_assignments:
-            role_string = role_assignment.role.name
-            context = role_assignment.get_context()
+        module_name, func_name = system_role_class.rsplit('.', 1)
```

Comment:
Is there a word that can be used instead of `class`, since we now will accept either a Django model class or a function? It would be nice to modify `settings.SYSTEM_WIDE_ROLE_CLASSES` and its use to better describe what it is and how it's used.